Readme/automation.md

```diff
@@ -0,0 +1,155 @@
+# Running Automation test with Kobiton with TeamCity
+## Table of contents
++ [A. Preparation](#a-preparation)
+  + [1. Prerequisites](#1-prerequisites)
+  + [2. Gettting Kobiton username and API key](#2-gettting-kobiton-username-and-api-key)
+  + [3. Samples](#3-samples)
++ [B. Setup](#b-setup)
+  + [1. Getting desired capabilities](#1-getting-desired-capabilities)
+  + [2. Configuring automation test script](#2-configuring-automation-test-script)
+  + [3. Configuring HockeyApp Webhooks listenner](#3-configuring-hockeyapp-webhooks-listenner)
+  + [4. Configuring TeamCity project](#4-configuring-teamcity-project)
++ [C. Execution](#c-execution)
+  + [1. Runnig automation test on Kobiton devices](#1-running-automation-test-on-kobiton-devices)
+  + [2. Fetch test session data through Kobiton REST API](#2-fetch-test-session-data-through-kobiton-rest-api)
++ [D. Feedback](#d-feedback)
+## A. PREPARATION
+### 1. Prerequisites
```

Comment:
- A server for receiving WebHooks:
+ Must be accessible from the Internet.
+ Has Ruby 2.x or higher installed.
- NodeJS 6.x or higher installed.

Reply:
.